includes/core/server/class-request.php

```diff
@@ -272,6 +342,24 @@ public function format_response( $response ) {
 		return new Response( $response );
 	}
 
+	/**
+	 * Used to validate passed ID from the constructor. You can easily override it if you need custom id handling in your classes.
+	 *
+	 * @param mixed $id ID parameter.
+	 *
+	 * @return void
+	 * @throws Invalid_Request_Parameter_Exception
+	 */
+	public function set_request_route_id_parameter( $id ) {
```

Comment:
This is a pretty critical method to ensure the integrity of all request classes. I think we should not expose it as `public`, instead limit it to `protected`. 
```suggestion
	protected function set_request_route_id_parameter( $id ) {
```

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/6234/commits/133739c69252d8698ca11e530c25b05dae0c2f6b